code/obj/machinery/mass_driver.dm

```diff
@@ -19,7 +19,7 @@
 	var/O_limit
 	var/atom/target = get_edge_target_turf(src, src.dir)
 	for(var/atom/movable/O in src.loc)
-		if(O.anchored || isobserver(O) || isintangible(O)) continue
+		if(O.anchored || isobserver(O) || isintangible(O) || isflockmob(O)) continue
```

Comment:
should use floating prop for this too

Reply:
https://github.com/stonepillars/goon-flock/pull/335